repos/system_upgrade/el7toel8/actors/storagescanner/libraries/storagescanner.py

```diff
@@ -236,9 +246,23 @@ def _get_systemd_mount_info():
 
 def get_storage_info():
     """ Collect multiple info about storage and return it """
+    fstab_entries, errors = _get_fstab_info('/etc/fstab')
+    for title, errs in errors.items():
+        # now multiple errors of the same type are grouped together, they are different only in remediation which
+        # points at the correct line
+        remediation = '\n'.join([e['remediation'] for e in errs])
+        reporting.create_report([
+            reporting.Title(title),
+            reporting.Summary(errs[0]['summary']),
+            reporting.Severity(errs[0]['severity']),
+            reporting.Tags([reporting.Tags.FILESYSTEM]),
+            reporting.Flags(errs[0]['flags']),
+            reporting.Remediation(hint=remediation),
+            reporting.RelatedResource('file', '/etc/fstab')
+        ])
```

Comment:
This logic is wrong from my POV. If I understand it well, it's exactly opposite of what we want. There is just one problem: invalid fstab. The number of wrong lines is just a parameter of the issue. And only thing we are interested about in this one particular actor is valid/invalid/readable fstab file.